topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py

```diff
@@ -291,7 +291,8 @@ def test_invalid_centroid_string() -> None:
     item = stac.Item(source_path)
     centroid = {"lat": "-41.28509", "lon": "174.77442"}
     metadata_loader_imagery_historic = MetadataLoaderImageryHistoric()
-    assert not metadata_loader_imagery_historic.is_valid_centroid(item, centroid)
+    assert not metadata_loader_imagery_historic.is_valid_centroid(item, centroid)  # type: ignore
```

Comment:
If this is causing only one type of issue you can be more specific in the comment, such as `# type: ignore[foo]` - "foo" being the name of the test printed by `mypy`. That way other mypy checks still run on the code.